common/src/main/java/com/viaversion/viaversion/protocols/v1_8to1_9/rewriter/EntityPacketRewriter1_9.java

```diff
@@ -302,11 +302,8 @@ public void register() {
                         }
                         properties.put(key, new Pair<>(value, modifiers));
                     }
-
-                    // == Why 15.9? ==
-                    // Higher values hides the cooldown but it bugs visual animation on hand
-                    // when removing item from hand with inventory gui
-                    properties.put("generic.attackSpeed", new Pair<>(15.9, ImmutableList.of( // Neutralize modifiers
+                    
+                    properties.put("generic.attackSpeed", new Pair<>(23.0, ImmutableList.of( // Neutralize modifiers
```

Comment:
```suggestion
                    properties.put("generic.attackSpeed", new Pair<>(20.0, ImmutableList.of( // Neutralize modifiers
```

Seems to be enough to remove the cooldown animation

Reply:
Were you holding a sword when testing? Cause the cooldown indicator didn’t disappear till I got to 23